optuna/trial.py

```diff
@@ -39,8 +39,8 @@ def suggest_discrete_uniform(self, name, low, high, q):
 
         raise NotImplementedError
 
-    def suggest_int(self, name, low, high):
-        # type: (str, int, int) -> int
+    def suggest_int(self, name, low, high, q):
```

Comment:
```suggestion
    def suggest_int(self, name, low, high, step=1):
```

IMO, we should specify the default value even if it's an abstract method.